packages/wasm-sdk/test/ui-automation/.env.example

```
@@ -0,0 +1,12 @@
+# Test Credentials for WASM SDK UI Tests
+# Copy this file to .env and fill with real values
+
+# Private keys for state transitions (DON'T STORE in production)
+TEST_PRIVATE_KEY_TRANSFER=YOUR_TRANSFER_PRIVATE_KEY_HERE
+TEST_PRIVATE_KEY_CONTRACT=YOUR_CONTRACT_PRIVATE_KEY_HERE
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The TEST_PRIVATE_KEY_CONTRACT key should go before the TEST_PRIVATE_KEY_TRANSFER key

(UnorderedKey)